{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "Spectrum Protect Backup-Archive Client",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "7.1.0.0"
                        },
                        {
                            "status": "affected",
                            "version": "7.1.8.6"
                        },
                        {
                            "status": "affected",
                            "version": "8.1.0.0"
                        },
                        {
                            "status": "affected",
                            "version": "8.1.8.0"
                        }
                    ]
                }
            ],
            "datePublic": "2019-11-22T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "IBM Spectrum Protect Backup-Archive Client 7.1 and 8.1 may be vulnerable to a denial of service attack due to a timing issue between client and server TCP/IP communications. IBM X-Force ID: 162477."
                }
            ],
            "metrics": [
                {
                    "cvssV3_0": {
                        "attackComplexity": "LOW",
                        "attackVector": "LOCAL",
                        "availabilityImpact": "HIGH",
                        "baseScore": 4.4,
                        "baseSeverity": "MEDIUM",
                        "confidentialityImpact": "NONE",
                        "exploitCodeMaturity": "UNPROVEN",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "HIGH",
                        "remediationLevel": "OFFICIAL_FIX",
                        "reportConfidence": "CONFIRMED",
                        "scope": "UNCHANGED",
                        "temporalScore": 3.9,
                        "temporalSeverity": "LOW",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.0/PR:H/AC:L/I:N/C:N/A:H/S:U/AV:L/UI:N/E:U/RC:C/RL:O",
                        "version": "3.0"
                    }
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "Denial of Service",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2019-11-25T17:01:24",
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://www.ibm.com/support/pages/node/1107777"
                },
                {
                    "name": "ibm-tsm-cve20194406-dos (162477)",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_XF"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/162477"
                }
            ]
        }
    },
    "cveMetadata": {
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "assignerShortName": "ibm",
        "cveId": "CVE-2019-4406",
        "dateReserved": "2019-01-03T00:00:00",
        "dateUpdated": "2019-11-25T17:01:24",
        "state": "PUBLISHED",
        "datePublished": "2019-11-25T17:01:24.440722Z"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}
